sphinxcontrib/openapi/openapi30.py

```diff
@@ -151,7 +152,7 @@ def _example(media_type_objects, method=None, endpoint=None, status=None,
         endpoint: The HTTP route to use in example.
         status: The HTTP status to use in example.
     """
-    indent = '   '
+    indent = '    '
```

Comment:
Is this necessary?

Reply:
This is not necessary but I find it more convenient to have the same indentation setting for both rst text and Python code. This is particularly helpful when both are included in the same source file, as this is often the case.

Reply:
That's fair but it means the diff is basically unreadable. I'm having a very hard time figuring out what's actually changed and what's just whitespace fixes.

Could you either (a) rebase these commits, squashing the functional changes in one or more commits and keeping the whitespace fixes in a separate commit or (b) revert the whitespace fixes and fix that via a separate PR? Without this, I'm going to have to manually do that myself just to see what you're trying to achieve :disappointed:

Reply:
This is now an option with the default set to 3. I reverted the changes due to an indentation of 4 (it was mostly in in the unit tests). This should help you analyze the changes.